driver-core/src/main/java/com/datastax/driver/core/RequestHandler.java

```diff
@@ -224,6 +225,16 @@ private void setFinalResult(
                 response.getCustomPayload());
       }
       callback.onSet(connection, response, info, statement, System.nanoTime() - startTime);
+      // if the response from the server has warnings, they'll be set on the ExecutionInfo. Log them
+      // here, if enabled.
+      if (logger.isWarnEnabled()
+          && response.warnings != null
+          && !response.warnings.isEmpty()
+          && Boolean.getBoolean(RequestHandler.LOG_REQUEST_WARNINGS_PROPERTY)) {
```

Comment:
What will happen if that property `RequestHandler.LOG_REQUEST_WARNINGS_PROPERTY))` will be not set

Reply:
From the Javadocs:
> If there is no property with the specified name, or if the specified name is empty or null, then false is returned.

So there shouldn't be any NullPointerException issues, and the current behavior is the default/unset behavior.